x/ibc/04-channel/keeper/packet.go

```diff
@@ -212,6 +212,24 @@ func (k Keeper) RecvPacket(
 		}
 	}
 
+	// check if the packet is being received in order
+	if channel.Ordering == types.ORDERED {
+		nextSequenceRecv, found := k.GetNextSequenceRecv(ctx, packet.GetDestPort(), packet.GetDestChannel())
+		if !found {
+			return sdkerrors.Wrapf(
```

Comment:
this line is not covered

Reply:
do you know of a way to see codecov locally on one's machine?